docs/resources/dcs_instance.md

```diff
@@ -4,66 +4,68 @@ subcategory: "Distributed Cache Service"
 
 # huaweicloud_dcs_instance
 
-Manages a DCS instance in the huaweicloud DCS Service.
+Manages a DCS instance within HuaweiCloud.
+
+!> **WARNING:** DCS for Memcached is about to become unavailable and is no longer sold in some regions.
+You can use DCS for Redis 4.0 or 5.0 instead. It is not possible to create Memcached instances through this resource.
+You can use this resource to manage Memcached instances that exist in HuaweiCloud.
 
 ## Example Usage
 
-### DCS Single Instance
+### Create a single mode Redis instance
 
 ```hcl
-data "huaweicloud_dcs_az" "az_1" {
-  code = "cn-north-1a"
-}
+variable az_code {}
```

Comment:
the variable name is confused, can we just set it to "cn-north-1a"?

Reply:
done